gobblin-yarn/src/main/java/gobblin/yarn/YarnService.java

```diff
@@ -434,6 +453,26 @@ private void handleContainerCompletion(ContainerStatus containerStatus) {
     this.eventBus.post(new NewContainerRequest(
         shouldStickToTheSameNode(containerStatus.getExitStatus()) ?
             Optional.of(completedContainerEntry.getKey()) : Optional.<Container>absent()));
+    return retryCount;
+  }
+
+  private Map<String, String> buildContainerStatusEventMetadata(ContainerStatus containerStatus, int retryCount) {
+    Map<String, String> eventMetaData = new HashMap<>();
+    eventMetaData.put(GobblinYarnMetricTagNames.CONTAINER_ID, containerStatus.getContainerId().toString());
+    eventMetaData.put(GobblinYarnEventConstants.EventTags.CONTAINER_STATUS_CONTAINER_STATE,
+        containerStatus.getState().toString());
+    if (ContainerExitStatus.INVALID != containerStatus.getExitStatus()) {
+      eventMetaData.put(GobblinYarnEventConstants.EventTags.CONTAINER_STATUS_EXIT_STATUS,
+          containerStatus.getExitStatus() + "");
```

Comment:
Just call `containerStatus.getExitStatus().toString()` instead of doing ` + ""`.